packages/glibc223.rb

```diff
@@ -3,17 +3,142 @@
 class Glibc223 < Package
   description 'GNU C Library'
   homepage 'https://www.gnu.org/software/libc/'
-  version '2.23-2'
+  version '2.23-3'
+  source_url 'https://ftp.gnu.org/gnu/glibc/glibc-2.23.tar.xz'
+  source_sha256 '94efeb00e4603c8546209cefb3e1a50a5315c86fa9b078b6fad758e187ce13e9'
+
   binary_url ({
-    aarch64: 'https://dl.bintray.com/chromebrew/chromebrew-cross/glibc223-2.23-2-chromeos-armv7l.tar.xz',
-     armv7l: 'https://dl.bintray.com/chromebrew/chromebrew-cross/glibc223-2.23-2-chromeos-armv7l.tar.xz',
-       i686: 'https://dl.bintray.com/chromebrew/chromebrew-cross/glibc223-2.23-2-chromeos-i686.tar.xz',
-     x86_64: 'https://dl.bintray.com/chromebrew/chromebrew-cross/glibc223-2.23-2-chromeos-x86_64.tar.xz',
+    aarch64: 'https://dl.bintray.com/chromebrew/chromebrew/glibc223-2.23-3-chromeos-armv7l.tar.xz',
+     armv7l: 'https://dl.bintray.com/chromebrew/chromebrew/glibc223-2.23-3-chromeos-armv7l.tar.xz',
+       i686: 'https://dl.bintray.com/chromebrew/chromebrew/glibc223-2.23-3-chromeos-i686.tar.xz',
+     x86_64: 'https://dl.bintray.com/chromebrew/chromebrew/glibc223-2.23-3-chromeos-x86_64.tar.xz',
   })
   binary_sha256 ({
-    aarch64: '34755443a25fbe74da7fd4c0d67f758aaf7078db76c45dfa36fdab86a3842266',
-     armv7l: '34755443a25fbe74da7fd4c0d67f758aaf7078db76c45dfa36fdab86a3842266',
-       i686: '14ce3a27fb989a7d26713ebe1c374b9c5f603f6570d99b015dc603223d055f63',
-     x86_64: '151470a91581b26c6b1eaf18aa085c2665aa16eccafb1435040983e1dd8acba0',
+    aarch64: 'de8de509b8296b73539a6ee06aefed5df9f3fa56e2d086addecc817f2eaeb1f2',
+     armv7l: 'de8de509b8296b73539a6ee06aefed5df9f3fa56e2d086addecc817f2eaeb1f2',
+       i686: 'f76de42054c1413d88618a218b9933056a0b31aaf6818043bee82a2ead245522',
+     x86_64: '9989a2208ede7055127780de0990d8bc6dd60379535c7d18a52da0742117bca9',
   })
+
+  depends_on 'libgd' => :build
+  depends_on 'texinfo' => :build
+
+  def self.patch
+    # Apply patch due to new version of binutils  which causes compilation failure
+    # http://lists.busybox.net/pipermail/buildroot/2017-August/199812.html
+    Dir.chdir "misc" do
+      if File.readlines("regexp.c").grep(/monitor/).any?
+        puts "Pachted!"
+      else
+        system "sed -i 's,char \\*loc1,char \\*loc1 __attribute__ ((nocommon)),g' regexp.c"
+        system "sed -i 's,char \\*loc2,char \\*loc2 __attribute__ ((nocommon)),g' regexp.c"
+        system "sed -i 's,char \\*locs,char \\*locs __attribute__ ((nocommon)),g' regexp.c"
+      end
+    end
+  end
+
+  def self.build
+    system "mkdir -pv glibc_build"
```

Comment:
Not sure if you are aware, but this works also:
```
Dir.mkdir "glibc_build"
```

Reply:
Can I keep it as it is?

Reply:
Sure, but just thought I would mention it in case you didn't know already.